sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/quickpulse/model/OpenTelMetric.java

```diff
@@ -0,0 +1,52 @@
+package com.azure.monitor.opentelemetry.exporter.implementation.quickpulse.model;
+
+import java.util.ArrayList;
+import java.time.LocalDateTime;
+
+public class OpenTelMetric {
+
+    private String name;
+    private ArrayList<OpenTelDataPoint> dataPoints;
+    private LocalDateTime lastTimestamp;
+
+    public OpenTelMetric(String name) {
+        this.name = name;
+        this.dataPoints = new ArrayList<>();
+        this.lastTimestamp = LocalDateTime.now();
+    }
+
+    public void addDataPoint(double value) {
+        OpenTelDataPoint dataPoint = new OpenTelDataPoint(value);
```

Comment:
Curious about why the value needs to be wrapped in an OpenTelDataPoint object. Is it possible for this.datapoints to be ArrayList<Double>?

Reply:
Each data point that is collected also comes with their associated dimensions, which need to also be stored. I am not storing them right now but will be doing so in the OTelDataPoint class once I move on to implementing filtering on dimensions.

Reply:
Interesting -  I didn't know filtering on the dimensions was in scope for your project.